tests/unit/test_dash.py

```diff
@@ -109,3 +112,70 @@ def test_kurtosis():
     kui = str(round(data['pnl'].kurtosis(), 2))
     assert kui == result['Value'].iloc[6]
```

Comment:
what is at index 6? why 6?

Reply:
value of kurtosis is in the 6th row

Reply:
can you put these to the function comment. I don't need to know, the readers of the code needs to know.